commitizen/cz/conventional_commits/conventional_commits.py

```diff
@@ -110,16 +109,14 @@ def questions(self) -> List[Dict[str, Any]]:
                 "name": "subject",
                 "filter": parse_subject,
                 "message": (
-                    "Subject. Concise description of the changes. "
-                    "Imperative, lower case and no final dot:\n"
+                    "Write a short and imperative summary of the code changes: (lower case and no period)\n"
```

Comment:
@Woile should we also put max length here?

Reply:
I don't think it's needed